core/shared/platform/windows/platform_internal.h

```diff
@@ -25,12 +25,17 @@
 #include <stdint.h>
 #include <malloc.h>
 #include <process.h>
+#include <winapifamily.h>
 #include <winsock2.h>
 #include <ws2tcpip.h>
 #include <windows.h>
 #include <basetsd.h>
 #include <signal.h>
 
+#if WASM_ENABLE_LIBC_WASI != 0 && WASM_ENABLE_UVWASI == 0
```

Comment:
Yes, whenever the uvwasi implementation is used, both are defined, https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/iwasm/libraries/libc-uvwasi/libc_uvwasi.cmake#L8. It's slightly confusing but I guess the flag 'WASM_ENABLE_LIBC_WASI' means libc WASI is enabled at all (regardless of the underlying implementation). There isn't any clash of symbols since we still only include one implementation, https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/build-scripts/runtime_lib.cmake#L92-L98.